packages/components/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### New Features
+
+-   `Popover`: added new `anchor` prop, supposed to supersede all previous anchor-related props (`anchorRef`, `anchorRect`, `getAnchorRect`). These older anchor-related props are now marked as deprecated and are scheduled to be removed in WordPress 6.3 ([#43691](https://github.com/WordPress/gutenberg/pull/43691)).
```

Comment:
Shouldn't these be under Breaking Changes?

Reply:
It depends on how we interpret this — technically, this PR doesn't introduce any breaking changes because the older props are still supported (they are just marked as deprecated, but they still work). The "breaking" will happen for WordPress 6.3, when those props will be removed.

But I'm not an expert of how we document these sort of changes in Gutenberg, and so I'll follow whatever you and other folks with more experience about this think is the best approach here (@talldan @ntsekouras @kevin940726 @bph )

Reply:
Going to merge this PR as-is, but I'm happy to make a change to the CHANGELOG in a follow-up if we think that this should have been marked as a breaking change.

Also, noting that if we think that this should have been marked as a breaking change, then we should mark as such also the changes that this PR makes to the `rich-text` package, and also some previous changes to the Popover component (like the deprecation of `__unstableShift`).

Reply:
I've been putting deprecations under Breaking Changes so I looked around, and found that we are all over the place 😅 Some packages have a ["Deprecations" heading](https://github.com/WordPress/gutenberg/blob/trunk/packages/blocks/CHANGELOG.md#11120-2022-07-13). I also found this [deprecations.md](https://github.com/WordPress/gutenberg/blob/trunk/docs/contributors/code/deprecations.md) file for the first time!

Given these options, I think I'd vote for putting deprecations under a Deprecations heading, and adding a Breaking Change at the point when the functionality is actually removed.

In any case I don't think we need to add a "New Features" heading because that's the same as an Enhancement, at least in semver terms.

Reply:
Alright! I will open a follow-up with those changes

Reply:
https://github.com/WordPress/gutenberg/pull/44195